internal/pkg/cli/dialog/use_template_test.go

```diff
@@ -57,9 +58,19 @@ func TestAskUseTemplateOptions(t *testing.T) {
 		assert.NoError(t, err)
 
 		time.Sleep(20 * time.Millisecond)
-		_, err = console.SendLine("text")
+		_, err = console.SendLine("text") // enter invalid string value
+		assert.NoError(t, err)
+
+		_, err = console.ExpectString(`Sorry, your reply was invalid: value "text" is not integer`)
+		assert.NoError(t, err)
+
+		time.Sleep(20 * time.Millisecond)
+		_, err = console.SendLine(strings.Repeat(Backspace, 4)) // remove "text"
```

Comment:
CLI predvyplni policko so starou hodnotou, potrebne zmazat.

Reply:
Este rozmyslam, ze tu je chyba, ma tam byt iba `Send`. ... takto sa odosle prazna hodnota ... a nahodou to funguje, ... lebo sa to opyta znova, ale uz tam nie je ziadna stara hodnota.

Reply:
Ahá, tak to vysvětluje jednu záhadu v logu testu. 😃